src/start_env/mod.rs

```diff
@@ -369,10 +375,15 @@ fn start_cnds(envfile_path: &PathBuf) -> impl Future<Item = Vec<Node<Cnd>>, Erro
 }
 
 fn temp_folder() -> PathBuf {
```

Comment:
:question: Couldn't this folder just live inside `path`?

Reply:
Yep, sounds good, I'll move it in my current PR :)